src/scss/_mixins.scss

```diff
@@ -101,15 +101,34 @@
     border-color: darken($color, $amount*2);
 }
 
-@mixin button($color: $c-brand-confirm, $amount:5, $icon:false) {
+@mixin button($color: $c-brand-confirm, $amount:5, $icon:false, $transparent:false, $border:0) {
     font-size: 1.4em;
     font-weight: 500;
-    border-width: 0;
+    border-width: $border;
     display: inline-block;
     padding: 1.2em 1em 0.9em;
     border-bottom: 3px solid;
     border-radius: 3px;
-    @include button-color($color, $amount);
+    @if $transparent {
+            background-color: transparentize($color, 1);
```

Comment:
Indentation seems weird here.

Yes, yes I know, a linter would have caught this. We're working on it. 😜

Reply:
Ah, yeah that was just a sloppy copy-paste on my part, pushed a fix!